pkg/service/healthcheck/service_integration_test.go

```diff
@@ -100,6 +105,7 @@ func TestStatus_Ping_Independent_From_REST_Integration(t *testing.T) {
 	}
 
 	// When #1 -> default scenario where everything works fine
+	// Retry 5 times, as the service could be stopped by other
```

Comment:
Where does this 5 come from?

Reply:
I don't know. Possibly some copy&paste leftover. Thanks.